bin/bugsnag-maze-runner

```diff
@@ -23,13 +33,38 @@ class MazeRunnerEntry
       opt :init, 'Initialises a new Maze Runner project'
 
       opt :version, 'Display Maze Runner and Cucumber versions'
-      opt OPTION_SEPARATE_SESSIONS, 'Start a new Appium session for each scenario', short: :none
+      opt 'separate-sessions', 'Start a new Appium session for each scenario',
+          short: :none,
+          type: :boolean,
+          default: false
+      opt OPTION_APP, 'The app to be installed and run against', short: :none, type: :string
+      opt OPTION_BS_LOCAL,
+          '(BS only) Path to the BrowserStackLocal binary',
+          short: :none,
+          type: :string,
+          default: '/BrowserStackLocal'
+      opt OPTION_FARM, 'Device farm to use: "bs" (BrowserStack) or "local"', short: :none, type: :string
```

Comment:
I wonder if it'd be simpler to have a switch for BrowserStack and a separate switch for local; not sure how many farms we want to support simultaneously so that might be a bad idea

e.g.

```sh
$ maze-runner --browserstack ...
$ maze-runner --local ...
$ maze-runner ... # equivalent to '--farm none'
```

Reply:
Hmm, a simpler command line and it's unlikely we will have more than a few different farms, but I think it would mean a more complex implementation?  Having to interrogate multiple options rather than a simple `case`, adding validation to reject mutually exclusive options being used together....

Reply:
I was thinking of something along the lines of

```ruby
config.farm = :local if options[OPTION_LOCAL]
config.farm = :bs if options[OPTION_BROWSERSTACK]
```

but yeah, if you want to validate the options it's just more work than the current method :p